requirements.txt

```diff
@@ -0,0 +1,3 @@
+# configparser==6.0.0
+# jinja==3.0.3
```

Comment:
part of the SDK image so commented out, but will be reinstated when updating Dockerfile python